src/core/processing-queue.js

```diff
@@ -0,0 +1,155 @@
+import config from "./config";
+import {
+  defined,
+  generateHash,
+  now,
+  objectType
+} from "./utilities";
```

Comment:
Good catch! We should probably turn on the eslint rule for indentation...and I should probably update my editor workspace to use tabs in QUnit.